scene/3d/vehicle_body.cpp

```diff
@@ -246,9 +258,15 @@ void VehicleWheel::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_friction_slip", "length"), &VehicleWheel::set_friction_slip);
 	ClassDB::bind_method(D_METHOD("get_friction_slip"), &VehicleWheel::get_friction_slip);
 
+	ClassDB::bind_method(_MD("is_in_contact"), &VehicleWheel::is_in_contact);
```

Comment:
This will fail building, as it should be using `D_METHOD` instead of `_MD` (see above).